datafusion/src/physical_plan/expressions/approx_median.rs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is this file needed anymore? I think it can be removed

Reply:
It is still needed as a wrapper for the expression.